### PR TITLE
Page header unification

### LINKED ISSUE #3014: Change "Edit Labels" to "Assign Labels"
### Is there an existing issue for this feature request?

- [X] I have searched the existing issues

### Is your feature request related to a problem?

Feature allows to assign label not edit. Editing labels is different feature.
screenshot_tx16s_23-01-11_12-27-22

### Describe the solution you'd like

Change menu option from "Edit Labels" to "Assign Labels"

### Describe alternatives you've considered

_No response_

### Additional context

_No response_

### LINKED ISSUE #3284: UI 2.9 Pages header unification
This issue is to unify header information which allow later (3.0) to convert it to proper breadcrumb (ie GROUP>PAGE>SUBPAGE). Follows https://github.com/EdgeTX/edgetx/pull/3238 
There are varoius fixes like
- Missing or wrong title
- Wrong letter type (capital vs small)

I think I've reviewed all pages but if you find inconsistency somwhere elsle please add in comment.

## RADIO SETTINGS

- [x] RADIO SETTINGS/TOOLS/Spectrum Analyser (sub page)
screenshot_tx16s_23-03-01_08-53-14
title1: TOOLS
title2: Spectrum Analyser

- [x] RADIO SETUP/Sound (sub page)
screenshot_tx16s_23-03-01_10-11-48
title1: RADIO SETUP
title2: Sound

- [x] RADIO SETUP/Variometer (sub page)
screenshot_tx16s_23-03-01_10-13-06
title1: RADIO SETUP
title2: Variometer

- [x] RADIO SETUP/Haptic (sub page)
screenshot_tx16s_23-03-01_10-13-35
title1: RADIO SETUP
title2: Haptic

- [x] RADIO SETUP/Alarms (sub page)
screenshot_tx16s_23-03-01_10-14-16
title1: RADIO SETUP
title2: Alarms
- [x] On RADIO SETUP page change button name from 'Alarm' to 'Alarms' to unify.

- [x] RADIO SETUP/Backlight (sub page)
screenshot_tx16s_23-03-01_10-16-04
title1: RADIO SETUP
title2: Backlight

- [x] RADIO SETUP/GPS (sub page)
screenshot_tx16s_23-03-01_10-16-44
title1: RADIO SETUP
title2: GPS

- [ ] HARDWARE/CALIBRATION (sub page)
screenshot_tx16s_23-03-01_11-09-53
title1: HARDWARE
title2: Calibration
This one requires to move information about current calibration step from header title2 variable to variable placed above graphical representation of sticks.

- [ ] HARDWARE/ANALOGS/Calibrated Analogs (sub page) - sub-pages navigation to change/discuss

- [ ] HARDWARE/ANALOGS/Filtered raw analogs with deviation (sub page) - sub-pages navigation to change/discuss

- [ ] HARDWARE/ANALOGS/Unfiltered raw analogs (sub page) - sub-pages navigation to change/discuss

- [ ] HARDWARE/ANALOGS/Min, max. and range (sub page) - sub-pages navigation to change/discuss

- [x] HARDWARE/Keys/SWITCHES TEST (sub page) - 2.8 version no change
screenshot_tx16s_23-03-01_10-23-03
title1: HARDWARE
title2: Keys

## MODEL SETTINGS

- [x] MODEL SETUP/Internal RF (sub page)
screenshot_tx16s_23-03-01_10-29-43
title1: MODEL SETUP
title2: Internal RF
 
- [x] MODEL SETUP/External RF (sub page)
screenshot_tx16s_23-03-01_10-30-45
title1: MODEL SETUP
title2: External RF
 
- [x] MODEL SETUP/Trainer(sub page)
screenshot_tx16s_23-03-01_10-31-23
title1: MODEL SETUP
title2: Trainer

- [x] MODEL SETUP/Timer 1-3 (sub page)
screenshot_tx16s_23-03-01_10-33-09
title1: MODEL SETUP
title2: Timer 1 (for other timer pages Timer 2 & Timer 3)

- [x] MODEL SETUP/Preflight Checks (sub page)
screenshot_tx16s_23-03-01_10-33-27
title1: MODEL SETUP
title2: Preflight Checks

- [x] MODEL SETUP/Trims (sub page)
screenshot_tx16s_23-03-01_10-33-49
title1: MODEL SETUP
title2: Trims

- [x] MODEL SETUP/Throttle (sub page)
screenshot_tx16s_23-03-01_10-34-13
title1: MODEL SETUP
title2: Throttle

- [x] FLIGHT MODE 0-8 (Flight mode edit page)
screenshot_tx16s_23-03-01_10-35-00
title1: FLIGHT MODES
title2: FM0 (for other pages FM1, FM2, FM3, FM4, FM5, FM6, FM7, FM8) 

- [x] MIXES (Mixes edit page1 & page2)
screenshot_tx16s_23-03-01_10-42-04
screenshot_tx16s_23-03-01_10-42-11
title1: MIXES
title2: CH1 (for ther pages CHxx) 

- [x] OUTPUTS (Output edit page) - page with no header
screenshot_tx16s_23-03-01_10-42-36
title1: OUTPUTS
title2: CH1 (for ther pages CHxx) 

- [x] CURVE (Curve edit page)
screenshot_tx16s_23-03-01_10-45-28
title1: CURVES (parent name: CURVES not as now CURVE)
title2: CV1 (for ther pages CVxx) 

- [x] GLOBAL VARIABLES/Global Variable (Global Variable edit page)
title1: GLOBAL VARIABLES (parent name: GLOBAL VARIABLES not as now Global Variable, Capital letters)
title2: GV1 (for ther pages GVxx) 

- [x] TELEMETRY/SENSOR (telemetry sensor edit page) 
screenshot_tx16s_23-03-01_10-55-53
title1: TELEMETRY
title2: SENSOR 1 (for ther pages SENSOR x) 

## MODEL MANAGER

- [x] Select Model (Models browsig)
screenshot_tx16s_23-03-01_10-58-42
title1: MODEL MANAGER
title2: Active _model_name_ (Active instead of Current) 

- [x] Select Model/Select a template folder (New Model creation) https://github.com/EdgeTX/edgetx/pull/3076
screenshot_tx16s_23-03-01_11-02-06
title1: MODEL MANGER
title2: New Model

## STATISTICS

- [ ] Statictics (sub page) - sub-pages navigation to change/discuss
- [ ] Debug (sub page) - sub-pages navigation to change/discuss

### LINKED ISSUE #3348: Misleading "RX channel order" on BW radio
### Is there an existing issue for this feature request?

- [X] I have searched the existing issues

### Is your feature request related to a problem?

The working used is misleading and technicaly not accurate imho. All this field does is define a default RADIO channel order, but actual RX channel order might be affected by things like multimodule channel reordering, or receiver side pin mapping on ELRS and others. In no way can it be guarented that RX will actually use the order shown here.

image

(Don't look at the RETT value, this is a WIP branch ;))

### Describe the solution you'd like

Remove RX from the wording

### Describe alternatives you've considered

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by pfeerick**
Resolves #3284
Resolves #3014
Resolves #3348
Resolves https://github.com/EdgeTX/edgetx/issues/3015

Summary of changes: 
 - Almost all mentioned pages have had their headers modified for consistency
- Exception: Hardware/calibration screen as I'm not adding a label to that since the entire calibration screen UI needs work
- Exception: Screens that have tab navigation, which should be brought up in a separate issue once there has been a proposal for changes to them.
- Also picks up "Edit Label" issue and changes to "Label model", which is consistent with other menu options
- Also picks up misleading "RX channel order" text on B&W radios

### CODE REVIEW

**Comment by pfeerick** [comment]
Can we get some translations the following new or changed strings please 😃 

- [ ] @zyren CN / TW
- [ ] @Eldenroot CZ
- [ ] @HThuren DA
- [ ] @ParkerEde / @TheIsotopes DE
- [ ] @Pat6874 FR
- [ ] @robustini IT
- [ ] @ToshihiroMakuuchi JP
- [ ] @ulfhedlund SE
- [ ] @ajjjjjjjj PL

`TR_DEF_CHAN_ORD` already existed as TR_DEF_CHAN_ORD, but the short form was incorrect, as the option only describes the default channel order, not the order RX channels will be in, hence the change. 

```
#define TR_DEF_CHAN_ORD TR("Def chan order", "Default channel order")

#define TR_MODEL_MANAGER "MODEL MANAGER"
#define TR_MAIN_MENU_MODEL_MANAGER "Model\nManager"

#define TR_ACTIVE "Active"
#define TR_LABEL_MODEL "Label model"
```


image
image
image

**Comment by robustini** [comment]
#define TR_DEF_CHAN_ORD TR("Ord. can. pred.", "Ordine canali predefinito")

#define TR_MODEL_MANAGER "GESTIONE MODELLI"
#define TR_MAIN_MENU_MODEL_MANAGER "Gestione\nModelli"

#define TR_ACTIVE "Attivo"
#define TR_LABEL_MODEL "Etichetta modello"

**Comment by ParkerEde** [comment]
DE
#define TR_DEF_CHAN_ORD TR("Std.Kanal Folge", "Std. Kanal Reihenfolge")

#define TR_MODEL_MANAGER "MODELL MANAGER"
#define TR_MAIN_MENU_MODEL_MANAGER "Modell\nManager"

#define TR_ACTIVE "Aktiv"
#define TR_LABEL_MODEL "Label zuordnen"

**Comment by pfeerick** [comment]
Before this goes any further - which should it be: 
1) "Models Manager"
2) "Model Manager" 
3) "Manage Models"

**Comment by philmoz** [comment]
> Before this goes any further with translations - which should it be:
> 
> 1. "Models Manager"
> 2. "Model Manager"
> 3. "Manage Models"

In English I think # 3 (Manage Models).
Number 1 just sounds wrong and number 2 is too close in meaning to 'Model Settings'.

**Comment by ToshihiroMakuuchi** [comment]
JP

#define TR_DEF_CHAN_ORD TR("Def chan order", "チャンネルマップ初期値")

#define TR_MODEL_MANAGER "モデル管理"
#define TR_MAIN_MENU_MODEL_MANAGER "モデル\n管理"

#define TR_ACTIVE "有効"
#define TR_LABEL_MODEL "モデルラベル"

**Comment by HThuren** [comment]
DA
#define TR_DEF_CHAN_ORD "Kanalrækkefølge"

#define TR_MODEL_MANAGER "Vælg Model"
#define TR_MAIN_MENU_MODEL_MANAGER "Vælg\nModel"

#define TR_ACTIVE "Aktiv"
#define TR_LABEL_MODEL "Navngiv model"

**Comment by ulfhedlund** [comment]
SE
#define TR_DEF_CHAN_ORD TR("Kanalordning","Förvald kanalordning")

#define TR_MODEL_MANAGER "MODELLHANTERING"
#define TR_MAIN_MENU_MODEL_MANAGER "Hantera\nmodell"

#define TR_ACTIVE "Aktiv"
#define TR_LABEL_MODEL "Sätt modelletikett"

**Comment by Eldenroot** [comment]
FOR CZE:

#define TR_DEF_CHAN_ORD TR("Def řaz kanálu", "Defaultní řazení kanálu")

#define TR_MODEL_MANAGER "NASTAVENÍ MODELU"
#define TR_MAIN_MENU_MODEL_MANAGER "Nastavení\nmodelu"

#define TR_ACTIVE "Aktivní"
#define TR_LABEL_MODEL "Štítek modelu"

**Comment by Pat6874** [comment]
sorry for my late reply, I was away

#define TR_DEF_CHAN_ORD TR("Ordre can. prédéf.", "Ordre canaux prédéfinis")

#define TR_MODEL_MANAGER "GESTION MODELES"
#define TR_MAIN_MENU_MODEL_MANAGER "Gestion\nModèles"

#define TR_ACTIVE "Actif"
#define TR_LABEL_MODEL "Etiquete modèle"

**Comment by Pat6874** [comment]
for the first menu, I suggest

[image: ecran1.jpg]

**Comment by Eldenroot** [comment]
No image displayed

**Comment by Pat6874** [comment]
[image: ecran1.jpg]

**Comment by Eldenroot** [comment]
@Pat6874 - no image displayed... you cannot send it via email :)

**Comment by Pat6874** [comment]
For the first menu:

Select Model --------------> CHOIX MODELE
Channel Monitor ----------> SCAN CANAUX
Model Settings ------------> REGLAGE MODELE
Radio Settings ------------> REGLAGE RADIO
Screens Settings --------> REGLAGE ECRAN
Reset Telemetry ---------> RAZ TELEMETRIE
Statistics -------------------> STATISTIQUES
About EDGE TX ---------> A PROPOS EDGE TX

**Comment by zyren** [comment]
****************************************************
CN

#define TR_DEF_CHAN_ORD TR("通道顺序", "默认通道顺序")

#define TR_MODEL_MANAGER "模型管理"
#define TR_MAIN_MENU_MODEL_MANAGER "模型\n管理"

#define TR_ACTIVE "当前模型"
#define TR_LABEL_MODEL "模型标签"
***************************************************
****************************************************
TW

#define TR_DEF_CHAN_ORD TR("通道順序", "默認通道順序")

#define TR_MODEL_MANAGER "模型管理"
#define TR_MAIN_MENU_MODEL_MANAGER "模型\n管理"

#define TR_ACTIVE "當前模型"
#define TR_LABEL_MODEL "模型標簽"
***************************************************

**Comment by pfeerick** [comment]
> Select Model --------------> CHOIX MODELE Channel Monitor ----------> SCAN CANAUX Model Settings ------------> REGLAGE MODELE Radio Settings ------------> REGLAGE RADIO Screens Settings --------> REGLAGE ECRAN Reset Telemetry ---------> RAZ TELEMETRIE Statistics -------------------> STATISTIQUES About EDGE TX ---------> A PROPOS EDGE TX

@Pat6874 I've not forgotten this comment. Is that the correct capitalisation for these strings?

```
#define TR_MAIN_MENU_SELECT_MODEL       "Select\nModel" // no longer used
#define TR_MAIN_MENU_MANAGE_MODELS      "Gestion\nModèles"
#define TR_MAIN_MENU_MODEL_NOTES        "Model\nNotes"
#define TR_MAIN_MENU_CHANNEL_MONITOR    "Channel\nMonitor"
#define TR_MAIN_MENU_MODEL_SETTINGS     "Model\nSettings"
#define TR_MAIN_MENU_RADIO_SETTINGS     "Radio\nSettings"
#define TR_MAIN_MENU_SCREEN_SETTINGS    "Screens\nSettings"
#define TR_MAIN_MENU_RESET_TELEMETRY    "Reset\nTelemetry"
#define TR_MAIN_MENU_STATISTICS         "Statistics"
#define TR_MAIN_MENU_ABOUT_EDGETX       "About\nEdgeTX"
```

**Comment by Pat6874** [comment]
Hello,
I have translated into french most of the terms of the fr.h file and l wish to send you everything.
How to do ?
Patrick 

⁣Télécharger BlueMail pour Android ​

Le 23 mars 2023, 02:58, à 02:58, Peter Feerick ***@***.***> a écrit:

**Comment by pfeerick** [comment]
You could either:
- fork this repository (if you haven't already, make a new branch based on main, make the changes there, and then submit that as a pull request
- upload the changed `fr.h` file here (zip the file, and then drag it to the comment area on github to attach it to to the message)
- send it to me on Discord... I'm pfeerick over there.

**Comment by Pat6874** [comment]
Hello,
I'm really bad at English and I don't understand how to use the github!
I am sending you the file as an attachment to this email.
Cordially
Patrick

Le ven. 31 mars 2023 à 02:23, Peter Feerick ***@***.***> a
écrit :

**Comment by Pat6874** [comment]
Dear Peter, i am not able to transfer my translated file.
Github is too complicate for me to understand.
I only know how to forward an attachement to an email.
Sorry.
If you have another solution to transfer my work, let me know.
Patrick 

⁣Télécharger BlueMail pour Android ​

Le 31 mars 2023, 02:23, à 02:23, Peter Feerick ***@***.***> a écrit:
